.storybook/decorators/GeneratedStory/GeneratedStory.styles.ts

```diff
@@ -0,0 +1,64 @@
+import { css } from '@leafygreen-ui/emotion';
+import { createUniqueClassName } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { typeScales } from '@leafygreen-ui/tokens';
+
+const indent = 16;
+
+export const generatedStoryWrapper = css`
+  /* padding: ${indent}px; */
```

Comment:
rm?

Reply:
I'm going to keep the variable to avoid merge conflicts (since I use it in the next PR), but I'll get rid of the contets

Reply:
I'm going to keep the variable to avoid merge conflicts (since I use it in the next PR), but I'll get rid of the rules